plugin-api/MetaflowPluginAPI.js

```diff
@@ -1,145 +1,193 @@
 const VERSION_INFO = {
-  api: '0.13.0'
-}
+  api: "0.14.0",
```

Comment:
this seems to be a breaking change update, can we do a semantic version?

Reply:
Yes i imagine we should up it to 1.0.0

Reply:
lets do it and I think we can merge 1.0.0